core/src/main/java/tech/tablesaw/api/BooleanColumn.java

```diff
@@ -447,15 +447,21 @@ public BooleanColumn set(int i, boolean b) {
     return this;
   }
 
-  private void set(int i, byte b) {
+  private BooleanColumn set(int i, byte b) {
     data.set(i, b);
+    return this;
   }
 
   @Override
   public BooleanColumn set(int i, Boolean val) {
     return set(i, val.booleanValue());
   }
 
+  @Override
+  public Column<Boolean> set(int row, String newValue, AbstractColumnParser<?> parser) {
+    return parser.isMissing(newValue) ? setMissing(row) : set(row, parser.parseByte(newValue));
```

Comment:
I make the comment before you noticed that parser actually handles missing values. So you could simplify this to just `return set(row, parser.parseByte(newValue));`

Reply:
Ok, I will simplify all methods to `return set(row, parser.parseByte(newValue));`.

Reply:
done.